_data/accessibility-tests/identifier.yml

```diff
@@ -0,0 +1,73 @@
+title: Identifier
+component_status: pass
+test_items:
+# General tests
+  - summary: Identifier appears consistently. 
+    summary_additional: The identifier appears in the same location across all pages of the website.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 3.2.3
+# Zoom/screen magnification tests
+  - summary: Identifier links are visible and functional when screen is magnified.
+    summary_additional: When you zoom in to 400%, identifier links are stacked and visible. No left-to-right scrolling is required.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.10
+  - summary: Identifier content is functional when screen is resized. 
+    summary_additional: When you zoom in to 200%, other content does not block or overlap the identifier content.
```

Comment:
Change additional copy to: When you zoom in to 200%, text remains readable and visible within the identifier.

Reply:
Confirmed!